src/views/components/injected/CourseCatalogInjectedPopup/GradeDistribution.tsx

```diff
@@ -211,8 +211,11 @@ export default function GradeDistribution({ course }: GradeDistributionProps): J
             {status === DataStatus.FOUND && (
                 <>
                     <div className='flex flex-wrap content-center items-center self-stretch justify-center gap-3'>
-                        <Text variant='small'>
-                            Grade Distribution for {course.department} {course.number}
+                        <Text variant='small' className='text-ut-black'>
+                            Grade Distribution for{' '}
+                            <Text variant='small' className='font-800!' as='strong'>
```

Comment:
my mistake, looks like there is a specific word that can be used instead of the number 800, and there are several instances of `font-thin`, `font-bold`, and `font-semibold` in the codebase
```suggestion
                            <Text variant='small' className='font-extrabold!' as='strong'>
```

Reply:
isn't 800 just font-bold though?

Reply:
![image](https://github.com/user-attachments/assets/325ee319-c3ae-4f40-8db5-b66d26b07d69)
https://tailwindcss.com/docs/font-weight